Mathlib/RingTheory/Localization/Basic.lean

```diff
@@ -546,6 +546,14 @@ theorem lift_injective_iff :
     Injective (lift hg : S → P) ↔ ∀ x y, algebraMap R S x = algebraMap R S y ↔ g x = g y :=
   (toLocalizationMap M S).lift_injective_iff hg
 
+variable (M) in
+include M in
+lemma injective_iff {T} [CommRing T] (f : S →+* T) :
```

Comment:
I think `IsLocalization.injective_iff` is a bit too generic for this. Maybe `injective_iff_map_algebraMap_eq` or something else that mentions `algebraMap`. Same for `injective_of`.